scripts/install_from_release.sh

```diff
@@ -1,9 +1,24 @@
+# Script to install scenarios file.  Pass in language code parameter for a particular language, such as it-it for Italian.
 set -e
 
+# Define the language parameter
+lang=${1:-''}
+
```

Comment:
NIT: I would set `SCENARIOS=""` at the top scope. Although all paths of the conditional declare the variable, I like to declare the variable in the outermost scope to ensure that it's declaration is always clear